### ISSUE TITLE
presets fail with "maximum stack depth exceeded"

### ISSUE THREAD

**Issue by ahknight** [AndrewRayCode/Bash-Prompt-Builder#52]
Tried the current site on Safari, Chrome, and Firefox with the same result.  Starting fresh and picking any preset spins the cursor for a bit and then fails.

**Comment by AndrewRayCode** [comment]
Yes, I'm sorry, I started a MooTools > jQuery refactor many moons ago but dropped the project for a while. I will look into reverting the site to the fully functional MooTools version.

**Comment by AndrewRayCode** [comment]
If you would like to test a fully working version of the function, you can copy what i have in my bashrc for now https://github.com/DelvarWorld/configs/blob/master/.bashrc#L371 up until the PS1 line
